yage-core/src/opengl/texture.rs

```diff
@@ -55,9 +55,9 @@ impl Texture {
     ///
     /// # Parameters
     /// - `context`: Active OpenGL context
-    /// - `unit`: Texture unit
+    /// - `unit`: Texture unit as GLenum, e.g. `glenum::TEXTURE0`
     ///
-    pub fn bind_active(&self, context: &Context, unit: u32) {
+    pub fn bind_active(&self, context: &Context, unit: glenum::types::GLenum) {
```

Comment:
I would like to revert this signature, so that Texture does not require the caller to care about GL_TEXTURE0 etc. It is clear from the semantics that we want to assign texture units, so why should I need to remember to use GL_TEXTURE0, and have the inconsistency with the shader value, which is just the number, not the enum.
So I would suggest to revert this and just add GL_TEXTURE0 here inside Texture.

Reply:
ok